src/main/target/common_pre.h

```diff
@@ -155,7 +155,7 @@
 #define USE_RX_SPI
 
 #define USE_RX_CC2500
-#define USE_RX_CRSF
```

Comment:
Why is this being removed ?

Reply:
Renamed to USE_RX_CRSF_BIND to make the function more obvious

Reply:
This is a little bit complicated.
There is a define `USE_RX_CRSF` which is always defined and is only used for`doBind` for CRSF serial receivers only when it and `SERIALRX_CRSF` are both true.  I'm guessing that this would apply also to any ELRS serial setup since it uses CRSF.

However there is a define or `USE_RX_EXPRESSLRS` which is used for SPI binding, and only when `RX_SPI_EXPRESSLRS` is also true.

It seems to me that `USE_RX_CRSF` exists to differentiate, in a genera sense, from `USE_RX_EXPRESSLRS`.  It just so happens at this point that these names are only used for binding purposes, but in future they may be required for  different things.
Hence I would leave this name unchanged.

Reply:
The PR that introduced the bind functionality for the CRSF protocol mistakenly introduced and used `USE_RX_CRSF` instead of the existing `USE_SERIALRX_CRSF`. The PR also missed a define to include the binding functionality when `USE_SERIALRX_CRSF` was chosen in the cloud build. That's why `bind_rx` was not available in the CLI for crossfire and expressLRS receivers.